src/test/data/installGuide/sample-projects/javascript-mocha-yarn2/yarn.lock

```diff
@@ -0,0 +1,17 @@
+# This file is generated by running "yarn install" inside your project.
+# Manual changes might be lost - proceed with caution!
+
+__metadata:
+  version: 4
+  cacheKey: 7
+
+"@yarnpkg/lockfile@npm:^1.0.0":
+  version: 1.1.0
+  resolution: "@yarnpkg/lockfile@npm:1.1.0"
+  checksum: 76f3f2625334fb4d37881408643b61ae08e1cfcdeb5db3e3e92f626b3535f99e963df8bafe5c07d85aa1dd09a308939eb4d0667ed4a4e93e645417b6af6e9a2f
+  languageName: node
+  linkType: hard
+
+"mocha@~10.2.0":
+  version "10.2.0"
```

Comment:
This file is kind of weird — it's neither valid YAML nor valid Yarn1 lockfile. Was that intended?

Reply:
Oh right, I forgot a colon `:` to make it valid YAML. I've now updated the fixture based on https://github.com/snyk/nodejs-lockfile-parser/blob/master/test/fixtures/goof/yarn2/yarn.lock, because I don't have a machine with yarn2 installed.